cognite/client/_api/files.py

```diff
@@ -744,17 +744,19 @@ def download(
         directory: Union[str, Path],
         id: Optional[Union[int, Sequence[int]]] = None,
         external_id: Optional[Union[str, Sequence[str]]] = None,
+        keep_folder_structure: bool = False,
     ) -> None:
         """`Download files by id or external id. <https://developer.cognite.com/api#tag/Files/operation/downloadLinks>`_
 
         This method will stream all files to disk, never keeping more than 2MB in memory per worker.
         The files will be stored in the provided directory using the name retrieved from the file metadata in CDF.
-
+        You can choose to keep the folder structure from CDF or not.
```

Comment:
Some more details would be good. Specify that the `directory` field will be respected for each file relative to the provided root directory.